src/status_im/chat/events/receive_message.cljs

```diff
@@ -78,16 +81,17 @@
                                                               current-account
                                                               (get-in fx [:db :chats chat-identifier])
                                                               contacts
-                                                              command)))]
+                                                              command)))
+            update-db-fx       #(-> %
+                                    (chat-utils/add-message-to-db chat-identifier chat-identifier enriched-message
+                                                                  (:new? enriched-message))
+                                    (unviewed-messages-model/add-unviewed-message chat-identifier message-id)
+                                    (assoc-in [:chats chat-identifier :last-message] enriched-message))]
         (cond-> (-> fx
```

Comment:
Seems like minor refactor, imo we should leave this alone for this type of PR since it makes it harder to review/can accidentally get bugs. Better do separate quick refactor PR

Reply:
OK to leave alone for now, no need to change for this PR

Reply:
it makes things more readbable which is kind of necessary for this function

Reply:
OK